docs/src/components/Demo.tsx

```diff
@@ -58,7 +58,7 @@ export const Demo = ({
         </Flex>
         <View
           flex="1"
-          maxWidth="50%"
```

Comment:
removing this maxWidth so that on mobile when the elements are stacked this can expand to 100% width